seda.muradyan/repo/exercise_10/exercise_10.py

```diff
@@ -0,0 +1,9 @@
+#home work Lesson 3 exe_10
+
+width = float(input("Enter your room width: "))
```

Comment:
RW: Include units in your prompt and output message; either feet or meters, depending on which unit you are more comfortable working with.